test/register/register.test.ts

```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+import { ModeHandler } from "../../src/mode/modeHandler";
+import { setupWorkspace, cleanUpWorkspace, assertEqualLines } from '../testUtils';
+
+suite("register", () => {
+
+  let modeHandler: ModeHandler;
+
+  setup(async () => {
```

Comment:
I would prefer these to be done in the style of newTest - I find that to be much easier to read. :)